src/index.html

```diff
@@ -13,6 +13,112 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+    </div>
+
+    <div class="stars stars--3">
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+    </div>
+
+    <div class="stars stars--4">
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+    </div>
+
+    <div class="stars stars--5">
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
```

Comment:
Don't use spaces in links. If these are meant to be anchor links, they should start with '#' followed by a meaningful identifier, without any spaces.